back/KinoFund/BLL/Movies/MoviesManager.cs

```diff
@@ -14,26 +19,40 @@ public class MoviesManager
     {
         private readonly MyContext _dbContext;
 
-        public MoviesManager(MyContext context)
+        private readonly RatingRepository _ratingRepo;
+
+        public MoviesManager(MyContext context, RatingRepository repository)
         {
             _dbContext = context;
-        }
+            _ratingRepo = repository;
 
+        }
+        
         public async Task<GetAllMoviesDTO> GetAllAsync()
         {
+            Dictionary<long, int> ratings = new Dictionary<long, int>();
```

Comment:
Remove it - you don't need it.
Use one cycle instead of two (Select - it's cycle too).
.....
var movieDtos = new List<MovieDto>
foreach(var movie in movies)
{
var rating = _repo.Get(); 
movieDtos.Add(movie.ToDto(rating)) // it will better to create extension method as in GetMovieInfo method

}
return ...